src/syntax_mapping.rs

```diff
@@ -1,9 +1,12 @@
 use std::path::Path;
 
 use crate::error::Result;
+use ignored_suffixes::IgnoredSuffixes;
 
 use globset::{Candidate, GlobBuilder, GlobMatcher};
 
+pub mod ignored_suffixes;
```

Comment:
I hope we don't need `pub(crate)` here since `syntax_mapping` module (parent) has it - it's not visible outside of this crate

Reply:
Yep, leaving this pub will still not expose it to outside of the crate as you say.